recipe/meta.yaml

```diff
@@ -38,14 +38,16 @@ package:
 
 source:
   url: https://github.com/kaldi-asr/kaldi/archive/{{ commit }}.tar.gz
-  sha256: 153ae41654046f1997c247d3328c17124745b70be090fe1ee73442160e7b08c7
+  sha256: aae30bb72ca2522eeec6582aaad092b4188fa17af33acb9a8d970419151edbd6
   patches:
-    - patches/0001-Update-cmake.patch
-    - patches/0002-Remove-Windows-executables.patch
+    #- patches/0001-Update-cmake.patch
+    #- patches/0002-Remove-Windows-executables.patch
+    - patches/0003-OpenFst-182-support.patch  # Patch based on https://github.com/kaldi-asr/kaldi/pull/4829
+    - patches/0004-Windows-shared-libs.patch
 
 build:
-  number: 4
-  skip: true  # [cuda_compiler_version == "10.2"]
+  number: 0
+  skip: true  # [cuda_compiler_version == "10.2" or (linux and cuda_compiler_version == "11.1")]
```

Comment:
Can we have a comment about the reason for this skip?

Reply:
Yes, so there was a weird error in this build: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=689618&view=logs&jobId=4f922444-fdfe-5dcf-b824-02f86439ef14&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=b2a8456a-fb11-5506-ca32-5ccd32538dc0

```
FAILED: src/cudafeat/CMakeFiles/cuda_compile_1.dir/cuda_compile_1_generated_feature-online-batched-cmvn-cuda-kernels.cu.o /home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/work/build/src/cudafeat/CMakeFiles/cuda_compile_1.dir/cuda_compile_1_generated_feature-online-batched-cmvn-cuda-kernels.cu.o 
cd /home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/work/build/src/cudafeat/CMakeFiles/cuda_compile_1.dir && /home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/_build_env/bin/cmake -E make_directory /home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/work/build/src/cudafeat/CMakeFiles/cuda_compile_1.dir//. && /home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/_build_env/bin/cmake -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=/home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/work/build/src/cudafeat/CMakeFiles/cuda_compile_1.dir//./cuda_compile_1_generated_feature-online-batched-cmvn-cuda-kernels.cu.o -D generated_cubin_file:STRING=/home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/work/build/src/cudafeat/CMakeFiles/cuda_compile_1.dir//./cuda_compile_1_generated_feature-online-batched-cmvn-cuda-kernels.cu.o.cubin.txt -P /home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/work/build/src/cudafeat/CMakeFiles/cuda_compile_1.dir//cuda_compile_1_generated_feature-online-batched-cmvn-cuda-kernels.cu.o.Release.cmake
/home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/_build_env/x86_64-conda-linux-gnu/include/c++/10.4.0/tuple(582): error: pack "_UElements" does not have the same number of elements as "_Elements"
          detected during:
            instantiation of "__nv_bool std::tuple<_Elements...>::__nothrow_constructible<_UElements...>() [with _Elements=<const int &>, _UElements=<>]" 
/home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/_build_env/x86_64-conda-linux-gnu/include/c++/10.4.0/bits/stl_map.h(502): here
            instantiation of "std::map<_Key, _Tp, _Compare, _Alloc>::mapped_type &std::map<_Key, _Tp, _Compare, _Alloc>::operator[](const std::map<_Key, _Tp, _Compare, _Alloc>::key_type &) [with _Key=int, _Tp=cub::CachingDeviceAllocator::TotalBytes, _Compare=std::less<int>, _Alloc=std::allocator<std::pair<const int, cub::CachingDeviceAllocator::TotalBytes>>]" 
/usr/local/cuda/include/cub/util_allocator.cuh(418): here

1 error detected in the compilation of "/home/conda/feedstock_root/build_artifacts/kaldi-split_1681415190237/work/src/cudafeat/feature-online-batched-cmvn-cuda-kernels.cu".
```

which led me to https://forums.developer.nvidia.com/t/nvc-20-9-fails-to-compile-code-instantiating-any-std-tuple-with-gcc-10-2-on-c-17/160987.  It popped up when I updated everything to use c++17, and it seems like it's specifically an issue with c++17 and 11.1, so I just disabled it.  Not sure if there's another workaround or solution.

Reply:
I don't mind disabling 11.1, which is getting long in the tooth anyway - I'd just like the context noted down in the recipe, so we have an easier time figuring out what we did in the future.